dtfs-central-api/test-helpers/get-sql-id-generator.ts

```diff
@@ -0,0 +1,11 @@
+/**
```

Comment:
```suggestion
/**
 * Generates a sequence of unique SQL IDs starting from 1.
 *
 * This function returns a generator that yields an incrementing number
 * each time it is called. It can be used to simulate the behavior of
 * an auto-incrementing SQL ID, where each call to the generator provides
 * the next ID in the sequence.
 *
 * @generator
 * @yields {number} The next unique SQL ID in the sequence.
 * @returns {number} The final value of the ID, although in typical use cases,
 * this generator function is intended to run indefinitely.
 *
 * @example
 * const idGenerator = getSqlIdGenerator();
 * console.log(idGenerator.next().value); // 1
 * console.log(idGenerator.next().value); // 2
 * console.log(idGenerator.next().value); // 3
 */
```